ext/hash/tests/hash-clone.phpt

```diff
@@ -301,7 +305,8 @@ string(16) "bebc746a33b6ab62"
 string(16) "893899e4415a920f"
 string(5) "joaat"
 string(8) "aaebf370"
-string(8) "513479b4"
+string(8) "836fb0e5"
```

Comment:
Why has the output of joaat changed here and elsewhere?

Reply:
Because joaat incorrectly computed hashes in which data was supplied piecemeal through multiple hash_update operations. It applied some “finalize” operations on every chunk of data.